src/actions/useGetUserFollows.tsx

```diff
@@ -0,0 +1,33 @@
+import { useState, useEffect, useContext } from 'react';
+import { db } from '../firebase';
+import { AuthContext } from '../context/AuthProvider';
+import { StateAble } from '../shared/interfaces/StateAbleInterface';
+
+interface UserFollows {
+  followed: string[];
+}
+
+export const useGetUserFollows = (): StateAble<string[]> => {
+  const currentUser = useContext(AuthContext);
+  const [state, setState] = useState<StateAble<string[]>>({
+    data: [],
```

Comment:
you could return userFollows because you get specific data (follows) , the same refers to useGetPosts, that's why the interface StateAble could be tough to define (and unnecessarry)

Reply:
I agree that StateAble interface is unnecessarry. Will delete it.